cmd/run_controller/main.go

```diff
@@ -76,8 +77,8 @@ func main() {
 	factory.StewardInformerFactory().Start(stopCh)
 	factory.TektonInformerFactory().Start(stopCh)
 
-	klog.V(2).Infof("Run controller")
-	if err = controller.Run(2, stopCh); err != nil {
+	klog.V(2).Infof("Run controller (%d threads)", threadiness)
```

Comment:
The controller doesn't start threads, but uses go routines, which get executed by a pool of threads managed by the Go runtime. Therefore the term "threadiness" is used instead of "threads".

```suggestion
	klog.V(2).Infof("Run controller (threadiness=%d)", threadiness)
```